### PR TITLE
added modal for confirmation on removing scaenrio

### PR DESCRIPTION
**Pull request by liorpr**

### CODE REVIEW

File: lib/ui/components/scenario-overview.js [hunk 1]
```diff
@@ -96,47 +104,68 @@ export class ScenarioOverview extends Component {
...
+
+            </div>
           </div>
-        </div>
-      </CardActions>
+        </CardActions>
+
+        <ConfirmRemoveScenario
+          open={ this.state.isConfirmRemoveModalOpen }
+          onClose={ this.closeConfirmRemoveModal.bind( this ) }
```
Comment (ilyagelman) [inline lib/ui/components/scenario-overview.js]: Binding in render is a bad practice, please bind in the constructor instead.

File: lib/ui/components/scenario-overview.js [hunk 2]
```diff
@@ -16,9 +16,17 @@ import EditIcon from 'material-ui/lib/svg-icons/editor/mode-edit';
 
 import { Request } from 'ui/components/request';
 import { ScenarioRequest } from 'ui/components/scenario-request';
+import { ConfirmRemoveScenario } from 'ui/components/modals/confirm-remove-scenario';
 
 export class ScenarioOverview extends Component {
 
+  constructor( props ) {
```
Comment (ilyagelman) [inline lib/ui/components/scenario-overview.js]: There should be no spaces inside parens according to the code style

File: lib/ui/components/scenario-overview.js [hunk 3]
```diff
@@ -96,47 +104,68 @@ export class ScenarioOverview extends Component {
...
+      </div>
     );
   }
+
+  openConfirmRemoveModal() {
+    this.setState( { isConfirmRemoveModalOpen : true } );
+  }
+
+  closeConfirmRemoveModal() {
+    this.setState( { isConfirmRemoveModalOpen : false } );
```
Comment (ilyagelman) [inline lib/ui/components/scenario-overview.js]: No need for spaces before colons

File: lib/ui/components/scenario-overview.js [hunk 4]
```diff
@@ -16,9 +16,18 @@ import EditIcon from 'material-ui/lib/svg-icons/editor/mode-edit';
 
 import { Request } from 'ui/components/request';
 import { ScenarioRequest } from 'ui/components/scenario-request';
+import { ConfirmRemoveScenario } from 'ui/components/modals/confirm-remove-scenario';
 
 export class ScenarioOverview extends Component {
 
+  constructor(props) {
+    super(props);
+    this.state = { isConfirmRemoveModalOpen : false };
```
Comment (morsdyce) [inline lib/ui/components/scenario-overview.js]: Add a line break after super. Remove space between property name and `:`

File: lib/ui/components/scenario-overview.js [hunk 5]
```diff
@@ -96,47 +105,68 @@ export class ScenarioOverview extends Component {
...
+        <ConfirmRemoveScenario
+          open={ this.state.isConfirmRemoveModalOpen }
+          onClose={ this.closeConfirmRemoveModal.bind( this ) }
+          onSubmit={ this.handleConfirmRemoveSubmit.bind( this ) }/>
+      </div>
     );
   }
+
+  openConfirmRemoveModal() {
+    this.setState({ isConfirmRemoveModalOpen : true });
```
Comment (morsdyce) [inline lib/ui/components/scenario-overview.js]: Remove space between property name and :

File: lib/ui/components/scenario-overview.js [hunk 6]
```diff
@@ -96,47 +105,68 @@ export class ScenarioOverview extends Component {
...
+  }
+
+  closeConfirmRemoveModal() {
+    this.setState({ isConfirmRemoveModalOpen : false });
```
Comment (morsdyce) [inline lib/ui/components/scenario-overview.js]: Remove space between property name and :

**Comment by morsdyce** [comment]
merged in #179

**Pull request closed without merging**